QualityUnit/postaffiliatepro-theme
# [890] Function "load_inline_svg" removed

**Changes proposed in this Pull Request**
Social icons in footer replaced with icons.svg.
Arrow in Affiliate Manager Cards block was replaced with icons.svg.
Function "load_inline_svg" removed.

**Testing instructions**
Check footer social icons.
Check arrow icon in Affiliate Manager Cards (e.g. [Affiliate Program Directory](https://www.postaffiliatepro.com/affiliate-program-directory/mobilecactus-affiliate-program/)).

**Checklist**

Close QualityUnit/web-issues#890
